src/js/pages/task-details/TaskDetail.js

```diff
@@ -20,6 +20,10 @@ const METHODS_TO_BIND = [
   'onTaskDirectoryStoreSuccess'
 ];
 
+const StatusMapping = {
+  'Running': 'service-running-state'
+};
```

Comment:
if we have this here and also inside of `SeriviceInfo`, maybe we should abstract it to a constants